src/screens/Main.tsx

```diff
@@ -39,50 +40,52 @@ export function Main({ className }: Props) {
 
   return (
     <>
```

Comment:
we can remove this since `BoxMain` enclose all

Reply:
done